typescript/ccip-server/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "ccip-server",
+    "version": "0.0.1",
+    "description": "CCIP server",
+    "typings": "dist/index.d.ts",
+    "typedocMain": "src/index.ts",
+    "files": [
+        "src"
+    ],
+    "engines": {
+        "node": ">=14"
+    },
+    "scripts": {
+        "start": "ts-node src/server.ts",
+        "dev": "nodemon src/server.ts",
+        "test": "jest"
+    },
+    "author": "brolee",
+    "license": "MIT",
```

Comment:
If possible (if this doesn't copy code already licensed MIT), please match the Apache license we use elsewhere

Reply:
Fixed.